src/pages/Checkout.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div id='checkout'>
+    Core Checkout
+  </div>
+</template>
+
+<script>
+import PersonalDetails from '../components/core/blocks/Checkout/PersonalDetails.vue'
```

Comment:
In new components we should use aliases from webpack

Reply:
OK